libsolidity/formal/SMTEncoder.h

```diff
@@ -71,6 +71,12 @@ class SMTEncoder: public ASTConstVisitor
 	/// otherwise _expr.
 	static Expression const* innermostTuple(Expression const& _expr);
 
+	/// @returns {_funCall.expression(), nullptr} if function call option
+	/// values are not given, and
+	/// {_funCall.expression().expression(), _funCall.expression()}
+	/// if they are.
+	static std::pair<Expression const*, FunctionCallOptions const*> functionCallExpression(FunctionCall const& _funCall);
```

Comment:
Why wrap the comment in an unnatural way like that if the code is longer anyway?
```suggestion
	/// @returns {_funCall.expression(), nullptr} if function call option values are not given, and
	/// {_funCall.expression().expression(), _funCall.expression()} if they are.
	static std::pair<Expression const*, FunctionCallOptions const*> functionCallExpression(FunctionCall const& _funCall);
```

Reply:
cus that was my editor's width limit